distribution/docs/src/main/resources/_contents/_sources/solrcloud-catalog-provider-contents.adoc

```diff
@@ -0,0 +1,85 @@
+==== Solr Cloud Catalog Provider
+
+Solr Cloud is a cluster of Solr Server instances that support fault tolerance and high availability.
+Each Solr Server instance in Solr Cloud provides a Solr Admin GUI.
+
+Synchronization between Solr Server instances is managed by Zookeeper.
+Zookeeper routes client requests to the proper Solr Server instance in the the Solr Cloud cluster.
+
+===== When to Use the Solr Cloud Catalog Provider
+
+If the ${branding} needs to be accessible for a higher than normal period with low instances of downtime, then Solr Cloud should be used.
+
+===== High Availability (Beta)
+
+High Availability in ${branding} is used when a highly available Solr service is needed.
+Solr Cloud is a highly available distributed capability of Solr.
+In Solr Cloud, if a Solr server fails, another Solr server will be able to provide distributed indexing and search capabilities to the ${branding}.
+Solr Cloud enables the ${branding} to be scalable and be fault tolerant.
+The instructions on setting up High Availabilty for ${branding} only include setup in a *NIX environment.
+
+[NOTE]
+====
+High Availability is currently in Beta version.
+It is implemented but has not been tested thoroughly.
+Bugs and feature changes may affect the final release.
+====
+
+===== Installation/Configuration Guide
+
+====== Prerequisites
+
+Before the ${ddf-solr} Application can be installed:
+
+* the ${branding} Kernel must be running,
+* the ${branding} Platform Application must be installed, and
+* the ${branding} Solr Application must be installed
+* ZooKeeper 3.4.5
+* NIX environment
+* JDK 8 or greater
+
+====== Installing Zookeeper
+
+Refer to https://zookeeper.apache.org/doc/r3.1.2/zookeeperStarted.html#sc_Download for installation instructions.
+
+[NOTE]
+====
+Minimum three Zookeeper nodes required.
+Three Zookeeper nodes are needed to form a quorum.
+A three Zookeeper ensemble allows for a single server to fail and the service will still be available.
+====
+
+====== Installing Solr Cloud
+
+Repeat the following procedure for each node that will be part of the Solr Cloud cluster:
+
+. Refer to https://cwiki.apache.org/confluence/display/solr/Apache+Solr+Reference+Guide for installation instructions.
+. Download jar files.
+The jars are needed to support geospatial and xpath queries and need to be installed on every "node" after the Solr Cloud installation instructions have been followed.
+
+.. http://artifacts.codice.org/service/local/repositories/releases/content/org/codice/thirdparty/jts/1.12_1/jts-1.12_1.jar
+.. http://artifacts.codice.org/service/local/artifact/maven/content?r=snapshots&g=ddf.platform.solr&a=solr-xpath&v=${ddf.version}
+
+. Copy downloaded jar files to: `/opt/solr/server/solr-webapp/webapp/WEB-INF/lib/`
+
+[NOTE]
+====
+Minimum two Solr Cloud instances required with each Solr Cloud instance having a two shard minimum.
```

Comment:
Should we just say "Solr Server instance" here instead of "Solr Cloud"?  Saying Solr Cloud makes it sound like we  are talking about two cloulds.